### ISSUE TITLE
Adding a channel half way through acquisition leads to an error (that is trapped)

### ISSUE THREAD

**Issue by raacampbell** [BaselLaserMouse/StitchIt#88]
```
Calculating final tilesFailed to collate. Reference to non-existent field 'layer'.                                                                             │·······································
fid =                                                                                                                                                          │·······································
                                                                                                                                                               │·······································
     3                                                                                                                                                         │·······································
         
```

So looks like the average collator needs work.

**Comment by raacampbell** [comment]
Also, when attempting to stitch all planes in channnel 2
```
 Stitching parameters:
Illumination correction: 1
Phase (comb) correction: 0
--------------------------------

ERROR: tileLoad is attempting to load channel 2 but this does not exist. Available channels: 3 
Error using tileLoad (line 94)
Output argument "index" (and maybe others) not assigned during call to "bakingtray/tileLoad".

Error in stitchSection (line 176)
parfor ii=1:size(section,1) %Tile loading is done in parallel, but it still seems faster to stitch in parallel
 
>>

**Comment by raacampbell** [comment]
Second issue fixed by: 4738e648d7db7987d4f60dd18c4d0ef2638dda78 and first issue fixed by: b4306c736bcff59ab8f6771d06ab8ba50e7f835a. These should both port just fine when we move to @ablot changes being discussed in Issue #83, so it's safe to close this.
